gammapy/stats/tests/test_feldman_cousins.py

```diff
@@ -0,0 +1,35 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from numpy.testing import assert_allclose
+from astropy.tests.helper import pytest
+
+from ...stats import (
+    find_confidence_interval_gauss,
+    find_confidence_interval_poisson,
+    construct_confidence_belt_PDFs,
+    get_upper_and_lower_limit,
+    fix_upper_and_lower_limit,
+    find_limit,
+    find_average_upper_limit,
+    construct_confidence_belt,
+)
+
+def test_significance_to_probability_normal():
```

Comment:
If you have "verified" outputs, e.g. either from the ROOT docs or tests (if they have any), or computed with ROOT, or from the FC paper, those are the most valuable.
In those cases, please add a docstring to the test function or a code comment mentioning the origin of the reference values.